docker/backend-dev/conf/Config.pm

```diff
@@ -269,90 +269,142 @@ XML
 
 # fields for which we will load taxonomies
 
-@taxonomy_fields = qw(states countries languages labels categories additives additives_classes allergens traces nutrient_levels misc);
+@taxonomy_fields = qw(states countries languages labels categories additives additives_classes
+vitamins minerals amino_acids nucleotides other_nutritional_substances allergens traces
+nutrient_levels misc ingredients ingredients_analysis nova_groups);
 
 
-# fields in product edit form
+# fields in product edit form, above ingredients and nutrition facts
 
-@product_fields = qw(quantity packaging brands categories labels origins manufacturing_places emb_codes link expiration_date purchase_places stores countries  );
+@product_fields = qw(quantity packaging brands categories labels origins manufacturing_places
+ emb_codes link expiration_date purchase_places stores countries  );
 
 # fields currently not shown in the default edit form, can be used in imports or advanced edit forms
 
 @product_other_fields = qw(
-producer_version_id
-net_weight_value net_weight_unit drained_weight_value drained_weight_unit volume_value volume_unit
-other_information conservation_conditions recycling_instructions_to_recycle recycling_instructions_to_discard
-nutrition_grade_fr_producer
+	producter_product_id
```

Comment:
I made a typo there, it's producer_product_id . It's fixed in https://github.com/openfoodfacts/openfoodfacts-server/pull/2093

Reply:
Fixed, thanks.